### ISSUE TITLE
Intellisense switched to case-sensitive compare

### ISSUE THREAD

**Issue by ninmonkey** [microsoft/vscode-powerquery#76]
## Problem

Autocomplete isn't completing unless case matches

### Expected Behavior

`table.c` should return `Table.Count`, `Table.Column`, etc...

### Actual Behavior

https://user-images.githubusercontent.com/3892031/113494445-cb89f080-94ad-11eb-974a-f70df6ffc898.mp4

### Tip

It seems to require the very first character to be capitalized, but the rest of it is case-insensitive. ex:

| Input | Successful Completion? |
|- | - |
|`table.c` | no|
|`Table.c` | yes|
|`TAbLE.C` | yes|

## Version

Tested on:
```
code = 1.54.3, 2b9aebd5354a3629c3aba0a5f5df49f43d6689f8, x64
PowerQuery.vscode-powerquery@0.1.18

code = 1.56.0-insider, 550035808407c15252e581e8abe1b820b9d311ee, x64
PowerQuery.vscode-powerquery@0.1.18
```

**Comment by ninmonkey** [comment]
[2021-04-03 - vscode-powerquery - case sensitivity intellisense.zip](https://github.com/microsoft/vscode-powerquery/files/6253780/2021-04-03.-.vscode-powerquery.-.case.sensitivity.intellisense.zip)

- One log uses `"powerquery.trace.server": "messages"`, the other `"powerquery.trace.server": "verbose"`
- The third log I made sure the `let-expression` was valid before logging

**Comment by JordanBoltonMN** [comment]
Resolved in #77
